crates/sui-graphql-rpc/Cargo.toml

```diff
@@ -72,6 +72,7 @@ expect-test.workspace = true
 hyper.workspace = true
 insta.workspace = true
 serde_json.workspace = true
+sui-framework.workspace = true
```

Comment:
Why is this needed?

Reply:
I added a test for query costing that needed some realistic objects to load into the DB and then fetch back out, so I decided to use the system packages!  (Note, this is a dev-dependency).

Reply:
Oh, I missed the part of it being a `dev-dependency`.